main.tf

```diff
@@ -253,5 +253,8 @@ module "dcos-lb" {
   name_prefix                        = "${var.name_prefix}"
   disable_masters                    = "${var.lb_disable_masters}"
   disable_public_agents              = "${var.lb_disable_public_agents}"
+  masters_acm_cert_arn               = "${var.masters_acm_cert_arn}"
+  masters_internal_acm_cert_arn      = "${var.masters_internal_acm_cert_arn}"
+  public_agents_acm_cert_arn         = "${var.public_agents_acm_cert_arn}"
```

Comment:
Cool I see this is also added here as well https://github.com/dcos-terraform/terraform-aws-lb-dcos/blob/release/v0.2/variables.tf#L56-L69

Reply:
And this version is tagged with `0.2.0` and is available for use. Just wanted to check. LGTM